benchmarks/apache-spark/src/main/scala/org/renaissance/apache/spark/DecTree.scala

```diff
@@ -2,52 +2,40 @@ package org.renaissance.apache.spark
 
 import java.nio.charset.StandardCharsets
 import java.nio.file.{Path, Paths}
-import java.util.zip.ZipInputStream
 
-import org.apache.commons.io.FileUtils
-import org.apache.commons.io.IOUtils
+import org.apache.commons.io.{FileUtils, IOUtils}
 import org.apache.spark.SparkContext
-import org.apache.spark.SparkConf
-import org.apache.spark.ml.Pipeline
-import org.apache.spark.ml.PipelineModel
-import org.apache.spark.ml.PipelineStage
-import org.apache.spark.ml.classification.DecisionTreeClassificationModel
-import org.apache.spark.ml.classification.DecisionTreeClassifier
-import org.apache.spark.ml.feature.StringIndexer
-import org.apache.spark.ml.feature.VectorIndexer
-import org.apache.spark.mllib.linalg.Vectors
-import org.apache.spark.mllib.regression.LabeledPoint
-import org.apache.spark.sql.DataFrame
-import org.apache.spark.sql.SQLContext
-import org.renaissance.Config
-import org.renaissance.License
-import org.renaissance.RenaissanceBenchmark
-
-import scala.util.Random
-
+import org.apache.spark.ml.{Pipeline, PipelineModel, PipelineStage}
+import org.apache.spark.ml.classification.{
```

Comment:
I'm personally not a fan of bulk imports, since I think that this should anyway be done by the IDE.
Does our `scalafmt` configuration like this?

Reply:
What do you mean by bulk? To me bulk would be anything that includes a `*` (in Java). The `{}` form in Scala list exact classes to import in less lines, so I don't really consider it bulk :-)

Scalafmt was fine with that.

Reply:
I mean that one-import-per-line instead of `{...}`. But, I leave it to you.

Reply:
This is what IDEA did when optimizing imports. You are right that with one-import-per-line it's easier to see diffs. Can scalafmt actually fix that?

Reply:
Maybe - perhaps @olafurpg could comment?

Reply:
For now, I have got rid of the `{}` imports manually, and will try to keep IDEA configured that way :-)

Reply:
There's a Scalafmt rewrite to expand grouped imports https://scalameta.org/scalafmt/docs/configuration.html#expandimportselectors More advanced "optimize imports" is out of scope since Scalafmt is only a syntactic tool